exchanges/order/order_test.go

```diff
@@ -934,6 +934,26 @@ func TestUpdateOrderFromModifyResponse(t *testing.T) {
 	assert.Nil(t, od.Trades)
 }
 
+func TestTimeInForceIs(t *testing.T) {
+	t.Parallel()
+	tifValuesMap := map[TimeInForce][]TimeInForce{
+		GoodTillCancel | PostOnly:    {GoodTillCancel, PostOnly},
+		GoodTillCancel:               {GoodTillCancel},
+		GoodTillCrossing | PostOnly:  {GoodTillCrossing, PostOnly},
+		ImmediateOrCancel | PostOnly: {ImmediateOrCancel, PostOnly},
```

Comment:
ImmediateOrCancel means its taker only and post only shouldn't work so can drop post only